setup.py

```diff
@@ -52,6 +52,6 @@
         'docs': docs_requires,
     },
     url='https://github.com/google/xarray-beam',
-    packages=setuptools.find_packages(),
+    packages=setuptools.find_packages(exclude=("examples")),
```

Comment:
I'm guessing this should probably be
```suggestion
    packages=setuptools.find_packages(exclude=["examples"]),
```
 
or maybe `exclude=('examples',)`